_layouts/staff.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+---
+
+<div class="staff">
+{% if site.staff %}
+	{% for person in site.staff %}
+		<article class="card">
+			{% if person.image %}
+			<img class="thumb" src="{{ person.image }}" frameborder="0"></img>
+			{% else %}
+			<img class="thumb" id="pokemon" frameborder="0"></img>
+			{% endif %}
+			<div class="text-wrapper">
+				<a class="title" href="mailto:{{ person.email }}">
```

Comment:
Should we have a [mail 📧 icon](https://fontawesome.com/icons/envelope?style=solid) indicating that the name is clickable and leads to their email?

Reply:
I personally feel like it's pretty obvious that the names are clickable?